packages/configurations/typescript.eslint-config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
       rules: {
         '@typescript-eslint/unbound-method': 0,
         '@typescript-eslint/no-namespace': 0,
-        'import/extensions': [2, 'always'],
+        'import/extensions': [2, 'ignorePackages'],
```

Comment:
let's remove this rule from both overrides and add it to top level config instead.
```javascript
module.exports = {
  extends: ['./eslint-config.js'],
  rules: {
    'import/extensions': [2, 'ignorePackages']
  },
  overrides: [
    ....
  ]
};
```

Reply:
Agreed and updated.